Description: get rid of sysctl includes on Linux
 The 'sysctl' syscall has been strongly deprecated on Linux for ages.
 Currently, on old architectures it will spam syslog whenever used, and on
 newer ones it's missing from the headers altogether.  Opencv has migrated
 away on Linux already, but #include statements were left lingering.
 .
 This patch removes such #includes on non-__APPLE__, unbreaking x32.
Author: Adam Borowski <kilobyte@angband.pl>

--- opencv-2.4.9.1+dfsg.orig/modules/core/src/parallel.cpp
+++ opencv-2.4.9.1+dfsg/modules/core/src/parallel.cpp
@@ -56,8 +56,8 @@
     #include <sys/types.h>
     #if defined ANDROID
         #include <sys/sysconf.h>
-    #elif !defined __GNU__
-        #include <sys/sysctl.h>
+     #elif defined __APPLE__
+         #include <sys/sysctl.h>
     #endif
 #endif
 
--- opencv-2.4.9.1+dfsg.orig/modules/core/src/system.cpp
+++ opencv-2.4.9.1+dfsg/modules/core/src/system.cpp
@@ -163,8 +163,6 @@ std::wstring GetTempFileNameWinRT(std::w
 #include <sys/types.h>
 #if defined ANDROID
 #include <sys/sysconf.h>
-#elif !defined __GNU__
-#include <sys/sysctl.h>
 #endif
 #endif
 
--- opencv-2.4.9.1+dfsg.orig/modules/highgui/src/cap_ffmpeg_impl.hpp
+++ opencv-2.4.9.1+dfsg/modules/highgui/src/cap_ffmpeg_impl.hpp
@@ -145,7 +145,7 @@ extern "C" {
 
 #if defined WIN32 || defined _WIN32
     #include <windows.h>
-#elif defined __linux__ || defined __APPLE__
+#elif defined __APPLE__
     #include <unistd.h>
     #include <stdio.h>
     #include <sys/types.h>
